vnext/Chakra/ChakraBytecodeStore.cpp

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ChakraBytecodeStore.h"
+#include "ChakraHelpers.h"
+#include "ChakraUtils.h"
+#include "Unicode.h"
+
+#pragma warning(push)
+#pragma warning(disable : 4244)
+#pragma warning(disable : 4245)
+#include <boost/crc.hpp>
+#pragma warning(pop)
+
+#include <cstdio>
+
+namespace facebook::react {
+
+///
+/// Metadata about a bytecode file that is persisted on disk
+///
+#pragma pack(push, 1)
+struct BytecodeFileHeader {
```

Comment:
We were missing this before, but I think we just lucked out in that the compiler didn't add any struct padding.

Reply:
For reference, this structure used to be called ByteCodePrefix in ChakraHelpers. The breaking format change is going from timestamp to checksum.